core/monitor.c

```diff
@@ -1991,6 +1991,7 @@ monitor_cache_enter(dcontext_t *dcontext, fragment_t *f)
                      * to last_copy, must re-clear!
                      */
                     md->last_fragment = NULL;
+                    KSTOP(trace_building);
```

Comment:
@derekbruening  Not sure if it is worth tracking down why this path is being hit now. Let me know.

Reply:
Weirdly this failure didn't show up on Github Actions for Ubuntu 18.04 #5072. But it did on my local workstation, so good to fix it anyway.

Reply:
Not worth further work.